packages/L10n/README.md

```diff
@@ -130,7 +124,44 @@ export default locales;
 
 Please do not using the same translation key as paprika's to avoid overriding. You can check them from here: https://github.com/acl-services/paprika/blob/master/packages/L10n/src/locales/en.yml
 
-##### Adding new translations
+## Interpolating components within a translation
+
+Imagine you have the string: `Processing failed with 3 errors. Click here to try again.` and you want to make the "Click here" substring a `<Button onClick={...}>` component. You can accomplish this with `@paprika/l10n` and react-i18next's [Trans component](https://react.i18next.com/latest/trans-component):
+
+```jsx
+// YourApp.js
+const Locales = {
+  en: {
+    translation: {
+      error: "Processing failed with {{count}} errors. <0>Click here</0> to try again.",
+      ...
+
+return (
+  <L10n locale="en" locales={Locales}>
+    <YourComponent />
+  </L10n>
+);
+```
+
+```jsx
+// YourComponent.js
+import { Trans } from "react-i18next";
+import useI18n from "@paprika/l10n/lib/useI18n";
+
+export default function YourComponent() {
+  const x = useI18n();
+  return (
+    <Trans i18nKey="error" i18n={x.i18n} count={3}>
+      <Button
+        kind="link"
+        onClick={...}
+      />
+    </Trans>
```

Comment:
Sorry I haven't got time to read the doc for `Trans` component, would it support multiple components in the translation content? e.g. `<0>Click here</0>` and then `<1>EXTRA info here</1>`